### PR TITLE
Issue #16 : Catch lateinit activity not initialized

### PR DESCRIPTION
**Pull request by jocelyngriselle**

### CODE REVIEW

File: android/src/main/kotlin/com/os/operando/advertisingid/AdvertisingIdPlugin.kt [hunk 1]
```diff
@@ -14,7 +14,7 @@ import io.flutter.plugin.common.PluginRegistry.Registrar
 import kotlin.concurrent.thread
 
 class AdvertisingIdPlugin() : FlutterPlugin, ActivityAware, MethodCallHandler {
-    private lateinit var activity: Activity
+    public lateinit var activity: Activity
```
Comment (operando) [inline android/src/main/kotlin/com/os/operando/advertisingid/AdvertisingIdPlugin.kt:17]: @jocelyngriselle 
Why did you change it to public?

Comment (jocelyngriselle) [reply android/src/main/kotlin/com/os/operando/advertisingid/AdvertisingIdPlugin.kt]: sorry no reason it's a typo

**Comment by operando** [comment]
Thanks!!
